systemd/ocp-cluster-ca.sh

```diff
@@ -53,10 +53,10 @@ updated_kubeconfig_path=/opt/crc/kubeconfig
 
 oc config set-credentials system:admin --client-certificate=${CLIENT_CA_FILE_PATH} --client-key=${CLIENT_CA_KEY_FILE_PATH} --embed-certs --kubeconfig="${updated_kubeconfig_path}"
 oc config set-context system:admin --cluster="${cluster_name}" --namespace=default --user=system:admin --kubeconfig="${updated_kubeconfig_path}"
-oc config set-cluster "${cluster_name}" --server="${apiserver_url}" --certificate-authority=$CA_FILE_PATH \
+oc config set-cluster "${cluster_name}" --server="${apiserver_url}" --insecure-skip-tls-verify=true \
     --kubeconfig="${updated_kubeconfig_path}" --embed-certs
 
-if oc get co --kubeconfig="${updated_kubeconfig_path}"; then
+if `oc get co --kubeconfig="${updated_kubeconfig_path}"`; then
```

Comment:
is this really required because `if oc get co --kubeconfig="${updated_kubeconfig_path}"` suppose to do same?